clients/unshaded/src/main/java/tachyon/client/WorkerBlockMasterClient.java

```diff
@@ -66,18 +66,18 @@ protected void afterConnect() {
    *
    * @param workerId the worker id committing the block
    * @param usedBytesOnTier the amount of used bytes on the tier the block is committing to
-   * @param tier the tier the block is being committed to
+   * @param tierAlias the alias of the tier the block is being committed to
```

Comment:
Yeah I was thinking of the StorageTier object as being the actual identity of the tier, and tierAlias is its name, and tierOrdinal is its position in the worker/master hierarchy. So a tier can be uniquely identified by its alias and ordinal.